dropwizard-logging/src/main/java/io/dropwizard/logging/FileAppenderFactory.java

```diff
@@ -260,21 +260,20 @@ public boolean isValidFileConfiguration() {
 
                 return appender;
             } else {
-                final TimeBasedFileNamingAndTriggeringPolicy<E> triggeringPolicy;
+                final TimeBasedFileNamingAndTriggeringPolicy<E> triggeringPolicy = new DefaultTimeBasedFileNamingAndTriggeringPolicy<>();
```

Comment:
I think it would be better to declare the triggering policy after the rolling policy has been initialized. Doing so, we keep it more close to its initialization and make the code more easy to understand for the reader.